src/networkInfo/selectors.ts

```diff
@@ -1,4 +1,7 @@
+import { SANCTIONED_COUNTRIES } from 'src/config'
 import { RootState } from 'src/redux/reducers'
 
 export const networkConnectedSelector = (state: RootState) => state.networkInfo.connected
 export const userLocationDataSelector = (state: RootState) => state.networkInfo.userLocationData
+export const userInSanctionedCountrySelector = (state: RootState) =>
```

Comment:
while this works, it's not optimal since this would be computed every time.

i would suggest to use `createSelector` because this handles memoisation for you. the selector can look something like https://github.com/valora-inc/wallet/blob/f5a649c0b6adc96a9f8895b5e9ccb748d555dcd7/src/localCurrency/selectors.ts#L72 and you can make use of the existing `userLocationDataSelector` :) hope this makes sense

Reply:
I just did what you said. Thanks for letting me know a more optimized implementation!